nebula/ui/components/inAppAuth/MZInAppAuthenticationThirdParty.qml

```diff
@@ -0,0 +1,132 @@
+        width: parent.width
+
+        RowLayout {
+            id: authHeader
+            Layout.preferredHeight: MZTheme.theme.rowHeight * 1.2
+            Layout.preferredWidth: parent.width
+            Layout.topMargin: MZTheme.theme.windowMargin / 2
+
+            MZLinkButton {
+                id: headerLink
+                objectName: _viewObjectName + "-getHelpLink"
+                Layout.alignment: Qt.AlignRight
+                Layout.rightMargin: MZTheme.theme.windowMargin
+                labelText: MZI18n.GetHelpLinkTitle
+                horizontalPadding: MZTheme.theme.windowMargin / 2
+                onClicked: {
+                    Glean.interaction.getHelpSelected.record({
+                        screen: _telemetryScreenId,
+                    })
+
+                    MZNavigator.requestScreen(VPN.ScreenGetHelp)
+                }
+            }
+        }
+
+        MZVerticalSpacer {
+            Layout.fillHeight: true
+            Layout.fillWidth: true
+            Layout.minimumHeight: MZTheme.theme.vSpacing / 2
+            Layout.maximumHeight: MZTheme.theme.vSpacing * 2
+        }
+
+        ColumnLayout {
+            spacing:  MZTheme.theme.windowMargin
+
+            Rectangle {
+                color: MZTheme.theme.transparent
+                Layout.preferredHeight: MZTheme.theme.rowHeight * 2
+                Layout.preferredWidth: 117
+                Layout.alignment: Qt.AlignHCenter
+
+                Image {
+                    id: img
+
+                    anchors.fill: parent
+                    sourceSize.height: MZTheme.theme.rowHeight * 2
+                    fillMode: Image.PreserveAspectFit
+                }
+            }
+
+            MZHeadline {
+                id: headline
+                objectName: _viewObjectName + "-headline"
+                width: undefined
+                Layout.fillWidth: true
+            }
+
+            ColumnLayout {
+                id: description
+                spacing:  MZTheme.theme.windowMargin
+                Layout.leftMargin: MZTheme.theme.vSpacing
+                Layout.rightMargin: MZTheme.theme.vSpacing
+                Layout.preferredWidth: col.width - MZTheme.theme.vSpacing * 2
+            }
+
+        }
+
+        MZVerticalSpacer {
+            Layout.fillHeight: true
+            Layout.fillWidth: true
+            Layout.minimumHeight: MZTheme.theme.vSpacing / 2
+        }
+
+        ColumnLayout {
+            id: footerContent
+            Layout.alignment: Qt.AlignBottom
+            Layout.maximumWidth: col.width - MZTheme.theme.vSpacing * 2
+            Layout.minimumWidth: col.width - MZTheme.theme.vSpacing * 2
+            Layout.leftMargin: MZTheme.theme.vSpacing
+            Layout.rightMargin: MZTheme.theme.vSpacing
+            Layout.bottomMargin: navbar.visible ? 0 : 34
+            spacing: 24
+
+            MZButton {
+                objectName: "buttonSignIn"
+                text: "Continue to sign in"
```

Comment:
this needs to go in strings.yaml

Reply:
this is marked as resolved, but a hardcoded string is still being used

Reply:
Sorry about this, not sure how that happened. Should be up to date now.